bfsdk/include/bfgsl.h

```diff
@@ -158,9 +158,7 @@ auto memset(span<DstElementType, DstExtent> dst, T val)
 
 #else
 
-#ifdef NEED_STD_LITE
-#include <bfstd.h>
```

Comment:
Why was this changed?

Reply:
Has to do with defining the vcpu_t types in the bftypes header. Can remove after moving those to a separate header.